common/OpTestUtil.py

```diff
@@ -1082,6 +1082,38 @@ def ping_fail_check(self, i_ip):
             return False
         return True
 
+    def check_pci_type(self, pci_device, cv_HOST):
+        self.bus_id = pci_device[0:7]
+        self.cmd = "cat /sys/class/pci_bus/{}/device/{}/uevent".format(self.bus_id, pci_device)
+        self.cmd_output = cv_HOST.host_run_command(self.cmd)
+        var = {}
+        for line in self.cmd_output:
+            if "=" in line:
+                name, value = line.split("=")
+                if name == "OF_TYPE":
+                    var[name] = str(value).rstrip()
+                    return var[name]
+
+    def map_pci_device_interface(self, pci_device, cv_HOST):
+        self.get_interface_cmd = "ls -la /sys/class/net/"
+        self.get_interface_cmd_output = cv_HOST.host_run_command(self.get_interface_cmd)
+        pattern = "{}".format(pci_device)
+        matching_lines = [line for line in self.get_interface_cmd_output if pattern in line]
+        interface = re.search(r'\benP\w+', str(matching_lines), re.I).group()
+        return interface
+
+    def configureIP_ping_test(self, interface, priv_IP, peer_IP, cv_HOST):
+        try:
+            self.ipconfig_cmd = "ifconfig {} {} netmask 255.255.255.0".format(interface, priv_IP)
```

Comment:
is ifconfig tool works fine for all the distro oses ? should we use IP commands ?

Reply:
Correct.  ifconfig tool deprecated and may not work on all distros . I modified to use  IP commands from iproute package